README.md

```diff
@@ -777,95 +712,24 @@ maintaining the Node.js project.
 Triagers follow the [Triage Guide](./doc/contributing/issues.md#triaging-a-bug-report) when
 responding to new issues.
 
-### Release keys
-
-Primary GPG keys for Node.js Releasers (some Releasers sign with subkeys):
-
-* **Antoine du Hamel** <<duhamelantoine1995@gmail.com>>
-  `5BE8A3F6C8A5C01D106C0AD820B1A390B168D356`
-* **Juan José Arboleda** <<soyjuanarbol@gmail.com>>
-  `DD792F5973C6DE52C432CBDAC77ABFA00DDBF2B7`
-* **Marco Ippolito** <<marcoippolito54@gmail.com>>
-  `CC68F5A3106FF448322E48ED27F5E38D5B0A215F`
-* **Michaël Zasso** <<targos@protonmail.com>>
-  `8FCCA13FEF1D0C2E91008E09770F7A9A5AE15600`
-* **Rafael Gonzaga** <<rafael.nunu@hotmail.com>>
-  `890C08DB8579162FEE0DF9DB8BEAB4DFCF555EF4`
-* **Richard Lau** <<richard.lau@ibm.com>>
-  `C82FA3AE1CBEDC6BE46B9360C43CEC45C17AB93C`
-* **Ruy Adorno** <<ruyadorno@hotmail.com>>
-  `108F52B48DB57BB0CC439B2997B01419BD92F80A`
-* **Ulises Gascón** <<ulisesgascongonzalez@gmail.com>>
-  `A363A499291CBBC940DD62E41F10027AF002F8B0`
-
-You can use the keyring the project maintains at
-<https://github.com/nodejs/release-keys/raw/refs/heads/main/gpg-only-active-keys/pubring.kbx>.
-Alternatively, you can import them from a public key server. Have in mind that
-the project cannot guarantee the availability of the server nor the keys on
-that server.
-
-```bash
-gpg --keyserver hkps://keys.openpgp.org --recv-keys 5BE8A3F6C8A5C01D106C0AD820B1A390B168D356 # Antoine du Hamel
-gpg --keyserver hkps://keys.openpgp.org --recv-keys DD792F5973C6DE52C432CBDAC77ABFA00DDBF2B7 # Juan José Arboleda
-gpg --keyserver hkps://keys.openpgp.org --recv-keys CC68F5A3106FF448322E48ED27F5E38D5B0A215F # Marco Ippolito
-gpg --keyserver hkps://keys.openpgp.org --recv-keys 8FCCA13FEF1D0C2E91008E09770F7A9A5AE15600 # Michaël Zasso
-gpg --keyserver hkps://keys.openpgp.org --recv-keys 890C08DB8579162FEE0DF9DB8BEAB4DFCF555EF4 # Rafael Gonzaga
-gpg --keyserver hkps://keys.openpgp.org --recv-keys C82FA3AE1CBEDC6BE46B9360C43CEC45C17AB93C # Richard Lau
-gpg --keyserver hkps://keys.openpgp.org --recv-keys 108F52B48DB57BB0CC439B2997B01419BD92F80A # Ruy Adorno
-gpg --keyserver hkps://keys.openpgp.org --recv-keys A363A499291CBBC940DD62E41F10027AF002F8B0 # Ulises Gascón
-```
```

Comment:
I don't think we can safely move this, there are many scripts out there that expect to find this in the README (including our own workflow in nodejs/release-keys)